Configure GraphQL Hello World

pedro-m-dietrich opened the pull request:
The setup allows queries with the browser at `localhost:4000/graphql` and with GraphQL at `localhost:4000`, using the `ruru` package.

pedro-m-dietrich committed:
Configure GraphQL Hello World

The setup allows queries with the browser at `localhost:4000/graphql`
and with GraphQL at `localhost:4000`, using the `ruru` package.

matheusflds commented on package.json:
```diff
@@ -0,0 +1,17 @@
+  "author": "Pedro Martins Dietrich",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.19.2",
+    "graphql": "^16.9.0",
+    "graphql-http": "^1.22.1",
+    "ruru": "^2.0.0-beta.13"
```
é uma boa fixar as versões (tirar o ^) para garantir que outras pessoas (ou ambientes de execução) venham a instalar a mesma versão da dependência em que que você desenvolveu e testou

pedro-m-dietrich committed:
Fix dependencies version

The `^` was removed to always install the same exact version as
specified in `package.json`.
